### PR TITLE
[Modular] Adds mouse waddle

### PR DESCRIPTION
**Pull request by ReturnToZender**
## About The Pull Request
Makes it so mice waddle

## Why It's Good For The Game
The mice need to waddle in sync with the rat king, and this provides that

## Changelog

:cl:
add: Mice waddle
/:cl:

### CODE REVIEW

**Comment by nevimer** [comment]
may be laggy

File: modular_zubbers/code/modules/mob/living/basic/vermin/mouse.dm [hunk 1]
```diff
@@ -0,0 +1,4 @@
+/mob/living/basic/mouse/Initialize(mapload, tame = FALSE, new_body_color)
+	.=..()
+	AddElement(/datum/element/waddling)
+//Bubber Edit Addition: MICE WADDLE NOW.
```
Comment (Majkl-J) [inline modular_zubbers/code/modules/mob/living/basic/vermin/mouse.dm:4]:
```suggestion
//Bubber Edit Addition: MICE WADDLE NOW.
/mob/living/basic/mouse/Initialize(mapload, tame = FALSE, new_body_color)
	. = ..()
	AddElement(/datum/element/waddling)
```

**Review by Majkl-J — CHANGES_REQUESTED** [review]
Simple formatting changes

**Comment by Majkl-J** [comment]
As for performance, I'll run checks for how long the respective movement procs take with and without the waddle. I know you can do this somehow, and I like challenging myself, so

**Comment by Majkl-J** [comment]
image

**Review by Majkl-J — APPROVED** [review]
Performance impact seems minor enough for me, Let's see what @nevimer says

**Pull request merged**